spec/spec_helper.cr

```diff
@@ -1,2 +1,4 @@
+ENV["CRYSTAL_ENV"] = "test"
+
```

Comment:
Not sure if this is the right way to do this, but I saw mention of the `CRYSTAL_ENV` environment variable [here](https://github.com/crystal-lang/crystal-env), so I borrowed it. :smile: